src/StackExchange.Redis.Extensions.Core/Implementations/RedisDatabase.cs

```diff
@@ -366,6 +366,19 @@ public async Task<bool> AddAllAsync<T>(IList<Tuple<string, T>> items, TimeSpan e
             return await Database.SetAddAsync(key, serializedObject, flag);
         }
 
+        public async Task<bool> SetContainsAsync<T>(string key, T item, CommandFlags flag = CommandFlags.None) where T : class
+        {
+            if (string.IsNullOrEmpty(key))
```

Comment:
maybe it makes sense to extract check and throwing exceptions to separate function
e.g. 
`ThrowIfInvalid<T>(string ket, T item)`

Reply:
Could make sense

Reply:
Ok, but let prepare change in separate PR